src/entity/EmpireHistory.ts

```diff
@@ -12,7 +12,7 @@ export default class EmpireHistory extends Model {
 
 	@Index()
 	@PrimaryGeneratedColumn()
-	empireHistory_id: number
+	id: number
```

Comment:
Made this change because the server kept failing to start due to another column in this entity with a conflicting name. Also, naming a table's primary key column `id` is a generally convenient convention to follow.